### PR TITLE
CNV-15707: RN for CentOS templates - Notable Tech Feature and Removed Feature

### PR DESCRIPTION
**Pull request by bgaydosrh**
For 4.10 Only

Notable technical changes

* The templates for CentOS Stream 8 and CentOS Stream 9 replace the CentOS Linux 8 template, which is no longer supported as of December 31st, 2021. The
templates for CentOS Stream 8 and CentOS Stream 9 are currently unsupported by {VirtProductName}.

JIRA: https://issues.redhat.com/browse/CNV-15707

Tagging @dominikholler for Code Review
Will tag a QE resource shortly.

Direct doc preview link:https://deploy-preview-42538--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-10-release-notes.html

### CODE REVIEW

File: virt/virt-4-10-release-notes.adoc [hunk 1]
```diff
@@ -77,6 +77,7 @@ The SVVP Certification applies to:
 
 //CNV-13680 Primary and secondary networks over a shared NIC on OVN clusters
 
+//CNV-15779 Network alerts when Kubemacpool detects duplicated MAC addresses
```
Comment (dominikholler) [inline virt/virt-4-10-release-notes.adoc:80]: @bgaydosrh Are you aware that there are multiple unrelated changes in this PR?

**Review by dominikholler — CHANGES_REQUESTED** [review]

File: virt/virt-4-10-release-notes.adoc (hunk 1 above)
Comment (bgaydosrh) [reply virt/virt-4-10-release-notes.adoc:80]: Yes, Those changes are getting my version of the notes rebased with the current version. This happens many times with release note updates. And Peer review will check for this too.

I'll tag Oded with a release notes to get a QE resource, assuming you are signing off.

Comment (bgaydosrh) [reply virt/virt-4-10-release-notes.adoc:80]: I will create another PR for this change, come to think of it, as the overall RN ownership has been transferred to another writer for this release (previously it was me). Thanks for reminder.

File: virt/virt-4-10-release-notes.adoc [hunk 2]
```diff
@@ -95,14 +96,22 @@ The SVVP Certification applies to:
...
+* The template for CentOS Linux 8 is no longer supported because support for CentOS Linux 8 is no longer supported as of December 31st, 2021. The template for CentOS Linux 8 has been replaced by templates for CentOS Stream 8 and CentOS Stream 9.
+
 //[id="virt-4-10-changes"]
-//== Notable technical changes
+== Notable technical changes
+
+//CNV-15707 Replacing template for CentOS Linux 8, because CentOS Linux 8 reached End Of Life (EOL) (Notable technical change)
+
+* The templates for CentOS Stream 8 and CentOS Stream 9 replace the CentOS Linux 8 template, which is no longer supported as of December 31st, 2021. The
+templates for CentOS Stream 8 and CentOS Stream 9 are currently unsupported by {VirtProductName}.
```
[the remark this answers is not in the document]
Comment (bgaydosrh) [reply virt/virt-4-10-release-notes.adoc:114]: Waiting on Ruth Netser per Jira comment.

[the remark this answers is not in the document]
Comment (rnetser) [reply virt/virt-4-10-release-notes.adoc:114]: CentsOS support is `community` level; if we mention that we support Fedora, we should include CentsOS as well, otherwise, I think we can drop it.

**Comment by bgaydosrh** [comment]
Closing this PR to replace with another: https://github.com/openshift/openshift-docs/pull/42909
